tensorflow/lite/kernels/pad_test.cc~

```diff
@@ -0,0 +1,786 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
This file should not be added. Looks like a backup file from the editor

Reply:
Sorry for my mistake